src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2.Test/ScenarioTests/IntegrationRuntimeTests.ps1

```diff
@@ -254,6 +264,12 @@ function Test-Shared-IntegrationRuntime
 
     try
     {
+        $test_mode = $Env:AZURE_TEST_MODE
```

Comment:
Wait-Seconds already has this functionality - it sets the sleep to 0 if the test mode is playback.  Please change back to just Wait-Seconds <num seconds>.

Edit: I will do this for you as this is the only change needed.